sentry-kotlin-compiler-plugin/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+    kotlin("jvm") version "1.8.20"
+    kotlin("kapt")
+    id("com.vanniktech.maven.publish") version "0.24.0"
+    id("org.jetbrains.compose") version "1.4.0"
+}
+
+repositories {
+    google()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+mavenPublishing {
+    // needs to match plugin-build/src/main/kotlin/io/sentry/SentryKotlinCompilerGradlePlugin.kt
+    coordinates("io.sentry", "sentry-kotlin-compiler-plugin", "1.0.0-mah-dev-026")
+}
+
+dependencies {
```

Comment:
`l`: would be great to extract versions/deps into `Dependencies.kt` to keep this aligned with the rest